src/plugin-config.js

```diff
@@ -29,7 +30,8 @@ function evaluateConfig(config) {
         updateSnapshotVersion: false,
         snapshotCommitMessage: 'chore: setting next snapshot version [skip ci]',
         debug: false,
-        mvnw: false
+        mvnw: false,
+        verboseMaven: false // default: do not echo maven commands
```

Comment:
```suggestion
        verboseMaven: false
```
The JSDoc already contains this information.